week2/whoareyou/main.js

```diff
@@ -0,0 +1,135 @@
+import pic1 from "./assets/김규민.jpeg";
+import pic2 from "./assets/전희선.jpeg";
+import pic3 from "./assets/서혜은.jpg";
+import pic4 from "./assets/황주희.jpeg";
+import pic5 from "./assets/백지연.png";
+
+const $ = (selector) => document.querySelector(selector);
+
+let currentStep = 0;
+
+const quizList = [
+  {
+    src: pic1,
+    answer: "김규민",
+  },
+  {
+    src: pic2,
+    answer: "전희선",
+  },
+  {
+    src: pic3,
+    answer: "서혜은",
+  },
+  {
+    src: pic4,
+    answer: "황주희",
+  },
+  {
+    src: pic5,
+    answer: "백지연",
+  },
+];
+
+function initGame({score, answer, image}){
+  currentStep = 0;
+  score.innerText = 0;
+
+  image.src = quizList[currentStep].src;
+}
+
+function showModal(modalContent, keepOpen){
+  const modal = $('.modal');
+  const modalBody = $('p.modal__body');
+  modalBody.innerText = modalContent;
+  modalBody.innerHTML = modalContent;
+  modal.classList.remove('hide');
+
+  if(keepOpen) return;
+  // setTimeout(()=>{
+  //   hideModal()
+  // }, 1500)
+  
+  //모달 바깥 클릭시 모달창 닫힘
+  modal.addEventListener('click',(e) => {
+    if(e.target.classList.value !== "modal__body"){
+      hideModal();
+    }
```

Comment:
이렇게 하면 모달의 modal__body 말고 다른 부분 클릭했을 때에도 모달창이 닫힐거여서 아래처럼 해주는게 확실할 것 같아요!
```suggestion
    if(e.target.closest('.modal')) hideModal();
```